src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const clone = { ...state };
+
+  for (const item of actions) {
+    if (item.type === 'addProperties') {
```

Comment:
`switch / case` instead of `if / else`.

Reply:
Thank you for review, but I like using switch/case only with return and when there are many conditions) I don't like break , I am afraid forgetting them)